src/components/Editor/Preview/Popup.js

```diff
@@ -237,9 +236,23 @@ export class Popup extends Component<Props> {
   }
 }
 
+const {
+  addExpression,
+  selectSourceURL,
+  selectSource,
+  loadObjectProperties,
+  openLink
+} = actions;
```

Comment:
it's very similar, but i think i would prefer

`import { addExpression, selectSourceURL } from actions`

Reply:
@jasonLaster Thanks for this. Yes this is more concise.

However, I encountered an issue that these imported action creators were undefined with this style, which is `openLink`, `loadObjectProperties` etc were undefined.

After looking into the transpiled codes, found:

- actions/index.js

Original codes
```js
export default Object.assign(
  {},
  navigation,
  breakpoints,
  ...
);
```
Transpiled codes
```js
Object.defineProperty(exports, "__esModule", {
  value: true
});
// ... ...
exports.default = Object.assign({}, navigation, breakpoints ...);
```

- Preview/index.js Case 1:

Original codes
```js
import actions from "../../../actions";
// ... ...
const {
  addExpression,
  loadObjectProperties,
  setPreview,
  clearPreview
} = actions;

export default connect(
  state => ({
    preview: getPreview(state),
    selectedSource: getSelectedSource(state),
    linesInScope: getInScopeLines(state),
    selectedFrameVisible: isSelectedFrameVisible(state)
  }),
  {
    addExpression,
    loadObjectProperties,
    setPreview,
    clearPreview
  }
)(Preview);
```
Transpiled codes:
```js
var _actions = __webpack_require__(18);
// _actions = { default: { __esModule: true, clearPreview, ... } }
var _actions2 = _interopRequireDefault(_actions);
// _actions2 == _actions
function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
//... ...
const {
  addExpression,
  loadObjectProperties,
  setPreview,
  clearPreview
} = _actions2.default;

exports.default = (0, _reactRedux.connect)(state => ({
  preview: (0, _selectors.getPreview)(state),
  selectedSource: (0, _selectors.getSelectedSource)(state),
  linesInScope: (0, _selectors.getInScopeLines)(state),
  selectedFrameVisible: (0, _selectors.isSelectedFrameVisible)(state)
}), {
  addExpression,
  loadObjectProperties,
  setPreview,
  clearPreview
})(Preview);
```

- Preview/index.js Case 2 - The broken case:

Original codes:
```js
import {
  addExpression,
  loadObjectProperties,
  setPreview,
  clearPreview
} from "../../../actions";

// ... ...

exports.default = (0, _reactRedux.connect)(state => ({
  preview: (0, _selectors.getPreview)(state),
  selectedSource: (0, _selectors.getSelectedSource)(state),
  linesInScope: (0, _selectors.getInScopeLines)(state),
  selectedFrameVisible: (0, _selectors.isSelectedFrameVisible)(state)
}), {
  addExpression,
  loadObjectProperties,
  setPreview,
  clearPreview
})(Preview);
```

Transpiled codes:
```js
var _actions = __webpack_require__(18);
// _actions = { default: { __esModule: true, clearPreview, ... } }

exports.default = (0, _reactRedux.connect)(state => ({
  preview: (0, _selectors.getPreview)(state),
  selectedSource: (0, _selectors.getSelectedSource)(state),
  linesInScope: (0, _selectors.getInScopeLines)(state),
  selectedFrameVisible: (0, _selectors.isSelectedFrameVisible)(state)
}), {
  // Below action creators are all undefined...
  addExpression: _actions.addExpression,
  loadObjectProperties: _actions.loadObjectProperties,
  setPreview: _actions.setPreview,
  clearPreview: _actions.clearPreview
})(Preview);
```

Looks like the diff between ES Module and NodeJS Require bothers the transpiler...